data_validation/cli_tools.py

```diff
@@ -1081,47 +1083,47 @@ def get_connection(connection_name):
     return mgr.get_connection_config(connection_name)
 
 
-def store_validation(validation_file_name, yaml_config):
-    """Store the validation YAML config under the given name."""
-    mgr = state_manager.StateManager()
-    mgr.create_validation_yaml(validation_file_name, yaml_config)
-
-
-def store_validation_json(validation_file_name, json_config):
-    """Store the validation JSON config under the given name."""
-    mgr = state_manager.StateManager()
-    mgr.create_validation_json(validation_file_name, json_config)
+def store_validation(validation_file_name, config, include_log=True):
+    """Store the validation config under the given name."""
+    validation_path = gcs_helper._get_validation_path(validation_file_name)
 
+    if validation_file_name.endswith(".yaml"):
+        config_str = dump(config, Dumper=Dumper)
+    elif validation_file_name.endswith("json"):
+        config_str = json.dumps(config)
+    else:
+        raise ValueError(f"Invalid validation file name: {validation_file_name}")
 
-def store_partition(target_file_path, yaml_config, target_folder_path=None):
-    """Store the partition YAML config under the given name."""
-    mgr = state_manager.StateManager(target_folder_path)
-    mgr.create_partition_yaml(target_file_path, yaml_config)
+    os.makedirs(os.path.dirname(validation_path), exist_ok=True)
+    gcs_helper.write_file(validation_path, config_str, include_log=include_log)
 
 
-def get_validation(validation_name, config_dir=None):
-    """Return validation YAML for a specific connection."""
+def get_validation(name: str, config_dir: str = None):
+    """Return validation YAML config."""
     if config_dir:
-        mgr = state_manager.StateManager(file_system_root_path=config_dir)
-        return mgr.get_validation_config(validation_name, config_dir)
+        validation_path = os.path.join(config_dir, name)
```

Comment:
Can a config dir also be in GCS?

Reply:
Yes! We already supported that before this PR though. That's why it was extra confusing - some commands supported direct GCS paths and others didn't 😅